test/Digipolis.Auth.UnitTests/PDP/PolicyDecisionProviderTests.cs

```diff
@@ -106,26 +103,25 @@ public async Task ApiKeyIsSetInHeader()
             };
 
             var mockHandler = new MockMessageHandler<PdpResponse>(HttpStatusCode.OK, pdpResponse);
-            var provider = new PolicyDecisionProvider(mockedCache.Object, Options.Create(_options), mockHandler, _logger);
+            var httpClient = new HttpClient(mockHandler);
+            httpClient.DefaultRequestHeaders.TryAddWithoutValidation(HeaderKeys.Apikey, _options.PdpApiKey);
```

Comment:
Is het toevoegen van de ApiKey aan de headers nodig in de headers? ApiKey wordt al meegegeven aan de constructor van de PolicyDecisionProvider via de options. Moet die header dan niet toegevoegd worden in de PolicyDecisionProvider zelf?

Dit gebeurt wel in de master branch: https://github.com/digipolisantwerp/auth_aspnetcore/blob/master/src/Digipolis.Auth/PDP/PolicyDescisionProvider.cs#L31

Reply:
We werken met de HttpClientFactory. Apikey wordt nu toegevoegd in ServiceCollectionExtensions (lijn 200)